json-to-pot.js

```diff
@@ -81,6 +81,22 @@ const findPath = (ob, key) => {
   return path.join('.')
 }
 
+const PARSED_VUE_FILES = getParsedVueFiles('./src/**/*.?(js|vue)')
+
+const BASE_URL = 'https://github.com/WordPress/openverse-frontend/blob/main'
+
+/**
+ * Returns the comment with a reference github link to the line where the
+ * string is used, if available. Example:
+ * #: https://github.com/WordPress/openverse-frontend/blob/main/src/components/HeroSection.vue#L6
+ * @param {string} keyPath (eg."hero.title")
+ * @return {string}
+ */
+const getRefComment = (keyPath) => {
+  const keyValue = PARSED_VUE_FILES.find((k) => k.path === keyPath)
+  return keyValue ? `\n#: ${BASE_URL}${keyValue.file}#L${keyValue.line}` : ''
```

Comment:
Could we remove the BASE_URL here, and instead use the pot format which is a relative file/line?

eg:`#: src/components/HeroSection.vue:6`

It looks like the leading `/` that is contained within `keyValue.file` will be fine to keep; eg once I applied the above change: https://translate.wordpress.org/projects/meta/openverse/es-co/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=12384826&filters%5Btranslation_id%5D=87424159

Reply:
```suggestion
  return keyValue ? `\n#: ${keyValue.file}#L${keyValue.line}` : ''
```